check.py

```diff
@@ -413,6 +413,7 @@ def fix(x):
     print s
     f = open('a.js', 'w')
     f.write(open(os.path.join(options.binaryen_bin, 'binaryen.js')).read())
+    f.write('var Binaryen = module.exports;\n')
```

Comment:
Can try. This depends on whether there is a `require` function with file system access in 'MozJS', respectively what else is the intended way to load CommonJS modules there. If that's possible, `a.js` could just `var Binaryen = require("./bin/binaryen.js");` the distribution instead of copying its contents.